pkgs/development/python-modules/pandoc-eqnos/default.nix

```
@@ -0,0 +1,27 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pandoc-xnos
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc-eqnos";
+  version = "2.5.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "tomduck";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-7GQdfGHhtQs6LZK+ZyMmcPSkoFfBWmATTMejMiFcS7Y=";
+  };
+
+  propagatedBuildInputs = [ pandoc-xnos ];
+
+  meta = with lib; {
+    description = "Pandoc-eqnos is a pandoc filter for numbering equations and their references when converting from markdown to other formats.";
```

```suggestion
    description = "Pandoc filter for numbering equations and their references";
```

see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes